resources/mysql/mysql.go

```diff
@@ -33,7 +33,7 @@ type Instance struct {
 	proc *os.Process
 }
 
-// Port returns the port of the mysql instance.
+// BackendPort returns the port of the mysql instance.
 func (i Instance) Port() int { return i.port }
```

Comment:
doesn't match

Reply:
It seems that Port is the function name.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

Should start with "Port returns xxx".